hokusai/production.yml

```diff
@@ -32,7 +32,21 @@ spec:
           env:
             - name: PORT
               value: "8080"
-            - name: TRACE_AGENT_HOSTNAME
+            - name: PUMA_WORKERS
+              value: "1"
+            - name: PUMA_THREAD_MIN
+              value: "5"
+            - name: PUMA_THREAD_MAX
+              value: "5"
+            - name: RAILS_SERVE_STATIC_FILES
+              value: 'true'
+            - name: RAILS_LOG_TO_STDOUT
+              value: 'true'
+            - name: PUMA_BIND
+              value: 'tcp://0.0.0.0:8080'
+            - name: MALLOC_ARENA_MAX
+              value: "2"
+            - name: DATADOG_TRACE_AGENT_HOSTNAME
```

Comment:
Surprised to see that this is unused. (Adding datadog can be a separate follow-up certainly.)